dependency.gradle

```diff
@@ -11,10 +11,31 @@ ext.versions = [
         rxJava: '2.0.0',
         rxLint: '1.0',
 
+        autoValue: '1.2',
```

Comment:
The plan would be any time a project updates to update here too. I can't think of an easy management method but I think that's something we'll work out as we go along.